core/mondoo-tls-security.mql.yaml

```
@@ -81,7 +81,7 @@ queries:
         notAfter - notBefore < 398*time.day
       }
   - uid: mondoo-tls-security-cert-no-cert-expired
-    title: Certificate has not expired
+    title: Certificate not near expiration or expired
```

This is a tough one. I think just adding a verb makes it fine.
```suggestion
    title: Certificate is not near expiration or expired
```